utils/msg.js

```diff
@@ -7,7 +7,7 @@ const DEFAULT_CHANNEL_ID = "1239383694033158254";
 /**
  * Use to send a simple string a channel,  default bot channel - social-credit-updates
  */
-export const msgChannel = (message, channelId = DEFAULT_CHANNEL_ID) => {
+export const msgChannel = (message, channelId = TEST_CHANNEL_ID) => {
```

Comment:
it will default to test channel... the whole point of default is that when you are testing you change the channel id to what you want and not changing the method